### ISSUE TITLE
[learning rate schedule]

### ISSUE THREAD

**Issue by jinggaizi** [flashlight/flashlight#556]
i saw the code shows a schedule that encoder layers and loss layer have different learning rate. i use pytorch and Adam with all layer ,then get a worse result, Is that because of a different schedule?

**Comment by jacobkahn** [comment]
@jinggaizi — it's hard to know exactly why training would differ across frameworks, but if you're talking about an ASR criterion, having a different learning rate was empirically beneficial for training.
